src/sentences/utils.rs

```diff
@@ -71,10 +71,35 @@ pub fn do_parse_lat_lon(i: &str) -> IResult<&str, (f64, f64)> {
     Ok((i, (lat, lon)))
 }
 
+/// Parses the variation between magnetic north and true north.
+/// The angle returned will be positive or negative depending on
+/// the East or West direction.<br>
+/// E.g:<br>
+/// "14.2,E" => 14.2 <br>
+/// "14.2,W" => -14.2 <br>
+pub fn do_parse_magnetic_variation(i: &str) -> IResult<&str, f64> {
+    let (i, variation_deg) = double(i)?;
```

Comment:
Let's use `float` here as well.
I believe that `f32` is much more accessible for embedded than `f64` and we should research whether we need `double` and how to allow different accuracy (f32, f64) with the parser.

Reply:
Yeah I used `f64` because I saw it was also used for the `parse_lat_lon` combinator in the same module. I'll switch to f32.